### DISCUSSION TITLE
Disable tenant menu with tenancy enabled

### DISCUSSION THREAD

**Discussion by pajkho** [filamentphp/filament#9709] [Help]
Is there any way to disable the tenant menu even though tenancy is enabled?

The reason is that I want the to make a custom tenant menu in a custom location. How to best achieve this?

**Comment by sinnbeck** [comment]
Maybe just set the to hidden?
https://filamentphp.com/docs/3.x/panels/tenancy#conditionally-hiding-tenant-menu-items

**Reply by pajkho** [reply]
That is only for specific menu items, not the whole menu.
